sleap_nn/data/sleap_dataset.py

```diff
@@ -0,0 +1,70 @@
+"""Handle distribution of sleap data."""
+from typing import Sequence, Tuple, Text, Union, Optional, List
```

Comment:
Some unnecessary imports here? Note that Python >= 3.9 doesn't need a lot of these (can just use `list` instead of `List` for type hinting)